lib/streams/contentHashStream.js

```diff
@@ -0,0 +1,38 @@
+const _ = require('lodash');
+const crypto = require('crypto');
+const through2 = require('through2');
+
+/*
+ * create a stream that generates a content-hash for each row
+ */
+
+function createContentHashStream() {
+  return through2.obj(function (record, enc, next) {
+    record.HASH = hash(record);
```

Comment:
so this would basically overwrite the existing `HASH` value from OA, while it continues to exist, right?

If so I really like it. It means the rest of the code can continue to use the value exactly as it expects :)

Reply:
yep, I figured it would be easier that way :)